src/static/js/pad_impexp.js

```diff
@@ -243,8 +243,19 @@ var padimpexp = (function()
       if (status !== "ok")
       {
         importFailed(status);
+      }else{
+        $('#import_export').removeClass('popup-show');
+      }
+
+      // directDatabaseAccess is a string not a boolean
+      if(directDatabaseAccess !== 'undefined'){
+        // commented out for now.  This needs a proper fix after 1.8.3
+        // TODO: pad.switchToPad() should not require a reload of the page.
+        // pad.switchToPad(clientVars.padId);
+        location.reload();
```

Comment:
This sidesteps the "two columns" bug.

How is it possible that `pad.switchToPad()` causes a duplication of the DOM nodes? @seballot work may have helped to uncover it, but I doubt this is just due to his recent work.

Can we investigate?

Reply:
I wrote the switchToPad logic maybe 8 years ago, I can look more into why it's duplicating but I'm worried about release Vs amount of time in day to get all these things done!  So this is a patch fix to ship 1.8.3 which causes very _little negative_ user impact as a trade off.

Reply:
I am ok with keeping the refresh _for now_. I want to get this out.

Can you assure me that the day after releasing **1.8.3** we can try to fix it?

Reply:
I can fix it before you release but don't merge it because I'd rather release something stable/guaranteed to work than something optimized.  Assurance made.